src/app/components/event-page/event-page.component.ts

```diff
@@ -117,23 +119,24 @@ export class EventPageComponent implements OnInit {
 		});
 	}
 
-  //to-do:
-  //remove mock data after demo, and use real data once the endpoints 
-  //for attendance are available
-  //isEventOwner should recieve a value after a check
-  
-	testArr: User[] = [
-		{ id: 1, name: 'Bill', email: 'abc', password: 'asdcasdcas' },
-		{ id: 2, name: 'Richard', email: 'abc', password: 'asdcasdcas' },
-		{ id: 3, name: 'Radahan', email: 'abc', password: 'asdcasdcas' },
-		{ id: 4, name: 'Godfrey', email: 'abc', password: 'asdcasdcas' },
-		{ id: 5, name: 'Michael', email: 'abc', password: 'asdcasdcas' },
-	];
-
 	openParticipantsDialog(): Observable<boolean> {
+		var eventParticipants: UserDetails[] = [];
+		var isHost: boolean;
+
+		this.participantsService.getParticipant(this.event.id).subscribe(
```

Comment:
in the ngOnInit we already have this.participantsArray initialized. You should make use of that.